addon/-private/data-view/radar/radar.js

```diff
@@ -224,7 +244,7 @@ export default class Radar {
   }
 
   afterUpdate() {
-    const { totalItems } = this;
+    const { _prevTotalItems: totalItems } = this;
```

Comment:
@runspired I did this to counter a very strange heisenbug that occurs in the scroll tests. Occasionally, somehow, `totalItems` will _change_ before this function is run. I believe this is only happening in PromiseArray scenarios, which makes sense, but I can't figure out _how_ - everything between the beginning on this update and now should all be run synchronously. How is an array being updated during a synchronous function sequence, with no breaks at all for async functions?

Reply:
Looking at it closer, it seems like this is happening because the promise is being resolved on the runloop, which makes sense. Seems strange that it would resolve on a nested runloop, but it seems to be happening

Reply:
🤔